massgov/mayflower
# Patternlab/DP-5950  accessibility tickets

## Description
low hanging fruit from the accessibility epic

## Related Issue / Ticket

- https://jira.mass.gov/browse/DP-9126
- https://jira.mass.gov/browse/DP-7874

## Steps to Test


1. Set focus on the callout link and see that the text is now underlined, like on hover.
2. Inspect the footer to see that the markup has been updated per the ticket request

## Screenshots
<img width="361" alt="screen shot 2019-02-14 at 3 37 48 pm" src="https://user-images.githubusercontent.com/18662734/52819868-10edc880-3070-11e9-830a-0eee327e9c49.png">
<img width="585" alt="screen shot 2019-02-14 at 3 48 36 pm" src="https://user-images.githubusercontent.com/18662734/52819874-14814f80-3070-11e9-9a2b-865ec3934eb6.png">
